test/integration/tests/bondingcurve.ts

```diff
@@ -47,7 +47,7 @@ describe('e2e-bondingcurve', function () {
     doLogging && console.log(`Environment loaded.`);
   });
 
-  describe('Reserve Stabilizer', async () => {
+  describe.skip('Reserve Stabilizer', async () => {
```

Comment:
We should unskip this (and fix any issues it has) until we actually ship the psm.